web-ui/src/main/resources/catalog/components/admin/uiconfig/partials/uiconfig.html

```diff
@@ -5,11 +5,10 @@
       <input type="checkbox"
              id="{{m}}-checkbox"
              data-ng-model="mCfg.enabled"/>&nbsp;
-      <label class="control-label"
-             for="{{m}}-checkbox">{{('ui-mod-' + m) | translate}}</label>
+      <h2 for="{{m}}-checkbox">{{('ui-mod-' + m) | translate}}</h2>
```

Comment:
`for` attribute on `h2` does not work so not sure if there is a workaround for that ?

Reply:
You are right, let me have a look